.changeset/weak-dingos-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": patch
```

Comment:
```suggestion
"@hashicorp/flight-icons": minor
```
I think adding a new feature to the pipeline like this would qualify as a minor release vs a patch which is typically for fixing something in pre-existing code